lib/IkiWiki/Plugin/ikistrap.pm

```diff
@@ -53,9 +53,9 @@ sub check($$) {
 sub refresh() {
 	return 0 unless($config{bootstrap_local});
 	mkdir("$config{destdir}/css");
-	mkdir("$config{destdir}/fonts");
+	mkdir("$config{destdir}/webfonts");
 	check("css/bootstrap.min.css", "https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css");
-	check("css/all.min.css", "https://unpkg.com/\@fortawesome/fontawesome-free\@5.10.1/css/all.min.css");
+	check("css/fontawesome.min.css", "https://unpkg.com/\@fortawesome/fontawesome-free\@5.10.1/css/all.min.css");
```

Comment:
This should be `css/font-awesome.min.css`

Reply:
Fixed. Sorry for the mistake :)